KitechRW/Kitech_backend
# Manager should be able to create services

#### what does this PR do?
- Allow the manager to create a service
#### Description of the task to be completed
- You are allowed to create a service after you log in
- Only a manager can create a service
#### How should this be manually tested
- Using postman call `/api/v1/services` endpoint
- Click send and once you are cleared with the errors you should be able to create a service
#### Any background context you want to provide
- N/A
#### What are the relevant Trello story link
- N/A
